CHANGES.rst

```diff
@@ -19,6 +19,9 @@ New features:
 Bug fixes:
 
 - ...
+- Fixed index error in cal.py when attempting to pop from an empty stack
+- Fixed type error in prop.py when attempting to join strings into a byte-string
+- Caught Wrong Date Format in ical_fuzzer to resolve fuzzing coverage blocker
```

Comment:
I think, if you split the PR into PRs for fixes that are independent and another one for the fuzzer code, some might be merged faster because it is very clear what it is about.

Reply:
okay! I can do that!